locations/spiders/homedepot.py

```diff
@@ -15,7 +17,12 @@ class HomeDepotSpider(SitemapSpider):
     ]
 
     def parse_store(self, response):
-        item = LinkedDataParser.parse(response, "LocalBusiness")
+        json_ld = json.loads(
+            response.xpath(
+                '//script[@id="thd-helmet__script--storeDetailStructuredLocalBusinessData"]/text()'
+            ).extract_first()
+        )
+        item = LinkedDataParser.parse_ld(json_ld)
         item["ref"] = item["website"].split("/")[-1]
         mapurl = urllib.parse.urlsplit(
             response.css('img[alt="map preview"]').attrib["src"]
```

Comment:
Is this a Google Maps URL? Should we expand `url_to_coords` to cover this URL format?

Reply:
Looks like it's a Bing URL. Here is a sample: 

```html
<img src="[https://dev.virtualearth.net/REST/v1/Imagery/Map/Road/38.59966,-90.487672/13?mapSize=60,60&amp;key=AoKTFnu3hdkRgVWl45fcxdz0WSDP6hroMDcPKXE4ZkQql3hPt3PtkCDwtNRCYgYA](https://dev.virtualearth.net/REST/v1/Imagery/Map/Road/38.59966,-90.487672/13?mapSize=60,60&key=AoKTFnu3hdkRgVWl45fcxdz0WSDP6hroMDcPKXE4ZkQql3hPt3PtkCDwtNRCYgYA)" alt="map preview" class="" height="60" width="60" loading="lazy"/>
```

Reply:
OK, I guess we'll come across it again, so I'd prefer it to be in `url_to_coords`. But it's not required.

Reply:
@Cj-Malone I added support for the Bing URLs to `url_to_coords`, would you mind reviewing?

Reply:
I think I'd have assumed it was always [8], but LGTM.